internal/permissions/permission_store.go

```diff
@@ -122,6 +127,15 @@ func (p *casbinStore) ListReadPermissionsByLexicon(owner string) (map[string][]s
 	return res, nil
 }
 
+// ListReadPermissionsByUser implements Store.
+func (p *casbinStore) ListReadPermissionsByUser(
+	owner string,
+	requester string,
+	nsid string,
+) ([]string, []string, error) {
+	panic("unimplemented")
```

Comment:
Lets avoid adding panics for unimplemented stuff. Can we bubble the error up and return 501 instead?